zeebe/exporters/opensearch-exporter/src/test/java/io/camunda/zeebe/exporter/opensearch/BulkIndexRequestTest.java

```diff
@@ -223,6 +255,72 @@ void shouldIndexRecordWithSequence() {
           .containsExactly(recordSequences.get(0).sequence(), recordSequences.get(1).sequence());
     }
 
+    @Test
+    void shouldIndexDeploymentRecordWithResourceMetadata() {
+      // given
+      final var value = new DeploymentRecord();
+      value
+          .resourceMetadata()
+          .add()
+          .setDeploymentKey(12345L)
+          .setResourceName("resourceName")
+          .setResourceId("resourceId")
+          .setChecksum(wrapString("checksum"))
+          .setResourceKey(1L)
+          .setVersion(1);
+      final var record =
+          recordFactory.generateRecord(
+              r -> r.withBrokerVersion(VersionUtil.getVersion()).withValue(value));
+
+      final var actions = List.of(new BulkIndexAction("index", "id", "routing"));
+
+      // when
+      request.index(actions.get(0), record, new RecordSequence(PARTITION_ID, 10));
+
+      // then
+      assertThat(request.bulkOperations())
+          .hasSize(1)
+          .map(operation -> MAPPER.readValue(operation.source(), MAP_TYPE_REFERENCE))
+          .extracting(source -> source.get("value"))
+          .extracting(source -> ((Map<String, Object>) source).get("resourceMetadata"))
+          .describedAs("Expect that the records are serialized with resourceMetadata")
+          .map(source -> MAPPER.writeValueAsString(source))
+          .containsExactly(
+              """
+              [{"checksum":"Y2hlY2tzdW0=","deploymentKey":12345,"duplicate":false,"resourceId":"resourceId","resourceKey":1,"resourceName":"resourceName","tenantId":"<default>","version":1,"versionTag":""}]""");
```

Comment:
```suggestion
[{"checksum":"Y2hlY2tzdW0=","deploymentKey":12345,"duplicate":false,"resourceId":"resourceId","resourceKey":1,"resourceName":"resourceName","tenantId":"<default>","version":1,"versionTag":""}]""");
```

Just to remove the blank line, not sure if the suggestion commit will fix it